### ISSUE TITLE
Ainedit does nothing

### ISSUE THREAD

**Issue by Marcgate** [nunuhara/xsystem4#3]
I'm trying to translate Oyako Rankan. The .ain file of the game is called "母娘乱館.ain", so I renamed it to "asd.ain". I used the following code in cmd.exe to get the txt file: "**aindump -t -o out.txt asd.ain**". I changed the recolletion mode's first title to random english characters, removed the ";" from the line, then saved the file and used the following command to write it to the .ain file: "**ainedit -t out.txt -o out.ain asd.ain**".
Then I renamed the "out.ain" to the original japanese name, and put it back to the game folder. The game started normally, but nothing has changed. I have tried other lines, but nothing changes. Also tried editing the code (out.jam), but that doesn't seem to work either.
What am I doing wrong?

**Comment by nunuhara** [comment]
The commands that you're using look right. Are the lines you're changing strings or messages? Strings are written as `s[number] = "text"` whereas messages are written as `m[number] = "text"`.
If it's a string, it might not be the string itself which the game is displaying. Sometimes the game will use a string as a key to look up some data in the .ex file, and then display the text stored in the .ex file. If that's the case, you would have to edit the .ex file rather than the .ain file (using the exdump/exbuild tools).

I did a quick test and I was able to modify the first line of text that appears when you start a new game (m[1973]), so I think the tools should be working.
![母娘乱館_20200804_205250](https://user-images.githubusercontent.com/52369405/89370795-71bef580-d696-11ea-93ef-1d6e0e4fe324.jpg)
